ckanext/benap/benap_schema.json

```diff
@@ -422,6 +423,27 @@
       "benap_helper_ontology": "NUTS1_BE",
       "required": "true"
     },
+    {
+      "field_name": "network_coverage",
+      "label": {
+        "en": "Network Coverage",
+        "fr": "Couveerture réseau",
+        "nl": "Netwerkdekking",
+        "de": "Netzabdeckung"
+      },
+      "display_snippet": "select.html",
+      "display_property": null,
+      "preset": "select",
+      "sorted_choices": "true",
+      "choices_helper": "benap_ontology_helper",
+      "benap_helper_ontology": "network_coverage",
+      "help_text":{
+        "en":"Network coverage describes the part of the transport network that is covered by the delivered content.",
+        "fr":"La couverture réseau décrit la partie du réseau de transport couverte par le contenu fourni",
+        "nl":"Netwerkdekking beschrijft het deel van het transportnetwerk dat door de geleverde inhoud wordt gedekt",
+        "de":"Netzabdeckung beschreibt den Teil des Verkehrsnetzes, der durch den gelieferten Inhalt abgedeckt wird."
+      }
+    },
```

Comment:
"validators": "benap_is_choice_null" toevoegen hier. Het is een optioneel veld dus als het leeg is moet die validator toegepast worden om de state op "deleted" te kunnen zetten. Consistentie! :)